recipe/meta.yaml

```diff
@@ -16,15 +16,21 @@ build:
 
 requirements:
   host:
-    - python >=3.7
+    - python >=3.8
     - pip
     - setuptools_scm
   run:
-    - python >=3.7
-    - dask
-    - zarr
-    - xarray
+    - python >=3.8
+    - zarr >=2.11
     - mypy_extensions
+    - dask
+    # dask[array]
+    - numpy >=1.21
+    # dask[diagnostics]
+    - bokeh >=2.4.2, <3
+    - jinja2 >=2.10.3
+    # extras but not all of the complete!?
+    - xarray >=2022.3
```

Comment:
@rabernat and @rsignell-usgs should we add all the "complete" extras ( apache_beam, fsspec, prefect<2, pyyaml, xarray>=2022.3) or should we remove xarray and keep just the minimum requirements?

Reply:
It seems strange to me to have workflow tools like `apache_beam` or `prefect` included but I'm just guessing.   I'll leave it to @rabernat .

Reply:
Those are all optional

Reply:
Yep. And the question is, should we add all the optional packages, just xarray as you had before here, all none (remove xarray). I'm not a rechunker user to know what is the best approach.